app/components/admin/edition_images/uploaded_images_component.html.erb

```diff
@@ -0,0 +1,90 @@
+<%= render "govuk_publishing_components/components/heading", {
+  text: "Uploaded images",
+  font_size: "l",
+} %>
+
+<% if can_have_lead_image? %>
+  <%= render "govuk_publishing_components/components/heading", {
+    text: "Lead image",
+    font_size: "m",
+    padding: true
+  } %>
+
+  <%= render "govuk_publishing_components/components/details", {
+        title: "Using a lead image",
+      } do %>
+    <p class="govuk-body">Using a lead image is optional. To use a lead image either select the
+    default image for your organisation or upload an image and select it as
+    the lead image.</p>
+    <p class="govuk-body">The lead image appears at the top of the document. It cannot be used
+    in the document body.</p>
+  <% end %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <% if lead_image %>
+    <div class="govuk-grid-column-one-third">
+      <img src="<%= lead_image[:url] %>" alt="<%= lead_image[:preview_alt_text] %>" class="app-view-edition-images__image">
+    </div>
+    <ul class="app-view-edition-images__details govuk-grid-column-two-thirds govuk-list govuk-list--bullet">
+      <li><strong>Caption: </strong><%= lead_image[:caption] %></li>
+      <li><strong>Alt text: </strong><%= lead_image[:alt_text] %></li>
+    </ul>
+  <% end %>
+
+  <% if lead_image || @edition.type == "CaseStudy" %>
+    <%= form_with(url: update_image_display_option_admin_edition_path(@edition), method: :patch) do |form| %>
+      <div class="app-view-edition-images__actions govuk-grid-column-full govuk-button-group">
+        <% if @edition.type == "CaseStudy" %>
+          <%= hidden_field_tag "edition[image_display_option]", new_image_display_option %>
+          <%= render "govuk_publishing_components/components/button", {
+            text: update_image_display_option_button_text,
+            secondary_solid: true,
+            margin_bottom: 4
+          } %>
+        <% end %>
+
+        <% if lead_image %>
+          <% lead_image[:links].each do |link| %><%= link %><% end %>
+        <% end %>
+      </div>
+    <% end %>
+  <% end %>
+
+</div>
+
+<%= render "govuk_publishing_components/components/heading", {
+  text: "Images available to use in document",
+  font_size: "m",
+  padding: true
+} %>
+
+<% if document_images %>
+  <%= render "govuk_publishing_components/components/inset_text", {
+    text: "Use the 'Insert image' button in the document tab to add images to body copy.",
```

Comment:
Sorry I didn't pick up on this sooner, but: isn't this undoing the change made in #7549?

We need to [avoid references to the 'Insert image' button](https://trello.com/c/vOZPG9Pt/1216-remove-references-to-insert-image-modal-because-its-not-part-of-release-16) because it isn't in this upcoming release.

Reply:
This PR went in first - the other was based on this so I think right now we're good!
https://github.com/alphagov/whitehall/blob/main/app/components/admin/edition_images/uploaded_images_component.html.erb#LL64C45-L64C45